packages/orchestrator/internal/sandbox/sandbox.go

```diff
@@ -772,7 +775,7 @@ func (s *Sandbox) Pause(
 	}, nil
 }
 
-func pauseProcessMemory(
+func (s *Sandbox) pauseProcessMemory(
```

Comment:
I would consider passing the cache dir here as a parameter. I think it was on purpose not placed on the Sandbox struct.